tools/gitlog2changelog.py.in

```diff
@@ -69,35 +86,35 @@ for line in fin:
         continue
     # Match the author line and extract the part we want
     # (Don't use startswith to allow Author override inside commit message.)
-    elif 'Author:' in line:
-        authorList = re.split(': ', line, 1)
+    elif "Author:" in line:
+        authorList = re.split(": ", line, 1)
         try:
             author = authorList[1]
-            author = author[0:len(author) - fin_chop]
+            author = author[0 : len(author) - fin_chop]
             authorFound = True
         except:
-            print ("Could not parse authorList = '%s'" % (line))
+            print("Could not parse authorList = '%s'" % (line))
 
     # Match the date line
-    elif line.startswith('Date:'):
-        dateList = re.split(':   ', line, 1)
+    elif line.startswith("Date:"):
+        dateList = re.split(":   ", line, 1)
         try:
             date = dateList[1]
-            date = date[0:len(date) - fin_chop]
+            date = date[0 : len(date) - fin_chop]
             dateFound = True
         except:
-            print ("Could not parse dateList = '%s'" % (line))
+            print("Could not parse dateList = '%s'" % (line))
     # The Fossil-IDs are ignored:
-    elif line.startswith('    Fossil-ID:') or line.startswith('    [[SVN:'):
+    elif line.startswith("    Fossil-ID:") or line.startswith("    [[SVN:"):
         continue
     # The svn-id lines are ignored
-    elif '    git-svn-id:' in line:
+    elif "    git-svn-id:" in line:
         continue
     # The sign off line is ignored too
-    elif 'Signed-off-by' in line:
+    elif "Signed-off-by" in line:
         continue
     # Extract the actual commit message for this commit
-    elif authorFound & dateFound & messageFound == False:
+    elif authorFound & dateFound & messageFound is False:
```

Comment:
Wondering if `&` here should better become `and`?..

Reply:
It was not suggested by either Black and Flake8

Reply:
Sure, I can add another commit to this that point.

Reply:
Ok, seems to generate same Changelog contents when using the same python interpreter with and without this change :)